session4/src/com/itschool/homework/Challenge3.java

```diff
@@ -0,0 +1,33 @@
+package com.itschool.homework;
+
+public class Challenge3 {
+
+    public static void main(String[] args) {
+
+        boolean a = true;
+        boolean b = false;
```

Comment:
_Write a Java program that takes two boolean inputs from a user_
`a` and `b` shouldn't be hard-coded with `true` and `false` but inserted by the user from keyboard, otherwise fine :)

Reply:
I do not know how to improve my code here :( Please. help me.

Reply:
Sure, here is how we can do it with the help of `Scanner` class that we studied it in the course:
        ```
        // creates a new Scanner object named "input" initialized with System.in source in constructor
        // (System.in stands for input system, that in our case is the keyboard)
        Scanner input = new Scanner(System.in);
        System.out.println("Please insert the first boolean: "); // ask the user to insert the first boolean value
        boolean a = input.nextBoolean(); // capture the next boolean input by user (at this step it is the first)

        System.out.println("Please insert the second boolean: "); // ask the user to insert the second boolean value
        boolean b = input.nextBoolean(); // capture the next boolean input by user (at this step it is the second one)
        // once both booleans captured, we can perform operations with them, 
        // so the rest of the code goes under this block of code
```